.github/workflows/test-short-suite.yml

```
@@ -0,0 +1,72 @@
+# This suite originally ran on 7 individual VMs that each took about 1m30s to execute
+# It's now combined onto a single VM that takes about 1m20s
+# The `if: always()` statements make all independent tests run, even if one fails.
+name: Test short suite
+
+on:
+  workflow_call:
+
+jobs:
+  test-short-suite:
```

```suggestion
  repository-health-checks:
```
I know these tests are grouped together because they're short, but perhaps we could have a more descriptive name along the lines of the above?

---

Yeah, that makes sense, done